src/encoded/schemas/changelogs/genetic_modification.md

```diff
@@ -2,6 +2,9 @@
 
 ### Minor changes since schema version 7
 
+* A combination of *introduced_elements* and *modified_site_nonspecific* can satisfy the *category* dependency for *insertion*.
```

Comment:
The code is out of date.  It needs to be brought up to current origin/dev and the demo rebuilt.  An associate wrangler should be able to help with this.

Reply:
I rebased the branch on top of current dev and made a new demo.
I know I need to squash the commits as well.

Reply:
I put ticket back to in progress.  We need to rebase again since we had updates in rc2.   Please rebuild the demo.  This is very important after a new release because the encoded and snovault deployment may have changed.

Don't worry about squashing.  It's nice todo on a brand new pr, but once it's been in review just add new commits.  Maybe we'll squash later.